app/controllers/schools/your_school_estates_controller.rb

```diff
@@ -27,27 +27,39 @@ def school_params
         :indicated_has_solar_panels,
         :indicated_has_storage_heaters,
         :has_swimming_pool,
-        :alternative_heating_oil,
-        :alternative_heating_lpg,
-        :alternative_heating_biomass,
-        :alternative_heating_district_heating,
-        :alternative_heating_ground_source_heat_pump,
-        :alternative_heating_air_source_heat_pump,
-        :alternative_heating_water_source_heat_pump,
-        :alternative_heating_oil_percent,
-        :alternative_heating_lpg_percent,
-        :alternative_heating_biomass_percent,
-        :alternative_heating_district_heating_percent,
-        :alternative_heating_ground_source_heat_pump_percent,
-        :alternative_heating_air_source_heat_pump_percent,
-        :alternative_heating_water_source_heat_pump_percent,
-        :alternative_heating_oil_notes,
-        :alternative_heating_lpg_notes,
-        :alternative_heating_biomass_notes,
-        :alternative_heating_district_heating_notes,
-        :alternative_heating_ground_source_heat_pump_notes,
-        :alternative_heating_air_source_heat_pump_notes,
-        :alternative_heating_water_source_heat_pump_notes
+        :heating_air_source_heat_pump_notes,
```

Comment:
Wondering if you could use the HEATING_TYPES here as well to define the three symbols for each parameter?